deploy/helm/sumologic/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sumologic
-version: 1.2.2
-appVersion: 1.2.2
+version: 1.3.0-beta.0
```

Comment:
Did we always start betas/alphas with `0` not with `1`?

Reply:
Yes.